p2p/discover/common.go

```diff
@@ -41,7 +41,7 @@ type Config struct {
 	PrivateKey *ecdsa.PrivateKey
 
 	// These settings are optional:
-	NetRestrict  *netutil.Netlist   // network whitelist
+	NetRestrict  *netutil.Netlist   // network restriction list
```

Comment:
This comment could be better. It's the list that nodes *are restricted to*. How about

```
// list of allowed IP networks
```